test/postprocess.test.js

```diff
@@ -122,7 +122,7 @@ describe("imagePostProcess", () => {
         assert.equal(receivedMetrics[0].callbackProcessingDuration + receivedMetrics[0].postProcessingDuration, receivedMetrics[0].processingDuration);
         assert.equal(receivedMetrics[1].eventType, "activation");
         assert.equal(receivedMetrics[1].callbackProcessingDuration + receivedMetrics[1].postProcessingDuration, receivedMetrics[1].processingDuration);
-    });
+    }).timeout(5000);
```

Comment:
What are the two versions? Which delegates do they use (is fftw there?)?
Did you observe the issue on circleci also when using the same imagemagick version as in Travis?

Reply:
CircleCI (`circleci/buildpack-deps:latest-dind`):

```
Version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 http://www.imagemagick.org
Copyright: © 1999-2017 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP 
Delegates (built-in): bzlib djvu fftw fontconfig freetype jbig jng jp2 jpeg lcms lqr ltdl lzma openexr pangocairo png tiff wmf x xml zlib
```

Travis CI (from [here](https://travis-ci.com/github/adobe/asset-compute-sdk/jobs/394515930#L7) under Build system information):

```
Version: ImageMagick 6.8.9-9 Q16 x86_64 2019-11-12 http://www.imagemagick.org
```

It's hard to change the versions on CircleCI since it's preinstalled on the base image and this test unfortunately relies on that one (unlike the `test-worker` tests which use our own custom docker image). This is just to unblock the frequent CI failures for now.